tests/__init__.py

```diff
@@ -86,6 +86,8 @@ def create_files(path: Union[str, Path], *contents: dict[str, Any]) -> None:
                     full_path.write_text(content)
                 elif isinstance(content, bytes):
                     full_path.write_bytes(content)
+                elif isinstance(content, list):
+                    full_path.write_text(yaml.safe_dump_all(content))
```

Comment:
This would make it counterintuitive to create yaml files containing a list at the root level since the only way to do it would be to pass the list as a tuple instead. Perhaps you should mention this behaviour in the docstring.

Reply:
Good point - I changed the multi-dock creation to use tuple instead as more specific (kept the old list behavior) and updated the docstring to reflect that.